### PR TITLE
Update contact journal screenshots (DEV)

### PR DESCRIPTION
**Pull request by jurajkusnier**
Update mock data for contact journal screenshots

### CODE REVIEW

**Comment by ralfgehrer** [comment]
Thx. If we were to enable the `firebase_screenshots` on this branch, we could easily validate the fix and won't need to build an RC.

**Review by ralfgehrer — DISMISSED** [review]
Many thanks. LGTM

**Comment by harambasicluka** [comment]
[Screenhots](https://65263-268027139-gh.circle-artifacts.com/0/zips/screenshots.zip)

File: .circleci/config.yml [hunk 1]
```diff
@@ -618,6 +618,7 @@ workflows:
   check_buildtype_device:
     jobs:
       - detekt
+      - firebase_screenshots
```
Comment (mtwalli) [inline .circleci/config.yml:621]: Please remove it?

Comment (harambasicluka) [reply .circleci/config.yml:621]: yes, as the screenshots are taken we can remove it :)

Comment (LukasLechnerDev) [reply .circleci/config.yml:621]: But the option to run the screenshots on firebase without modifying the config.yml file for certain branches/PRs would be nice. Any ideas about how we could accomplish this?

Comment (ralfgehrer) [reply .circleci/config.yml]: done

**Review by ralfgehrer — APPROVED** [review]

**Review by LukasLechnerDev — APPROVED** [review]
The correct text "Diese müssen nicht in Zusammenhang mit den von Ihnen erfassten Personen und Orten stehen." is now added to a high or low risk in our contact journal, so LGTM!

File: .circleci/config.yml (hunk 1 above)
Comment (mtwalli) [reply .circleci/config.yml]: It is a nice idea, I don't have any on top of my head. if anyone has ideas let me know?

**Review by mtwalli — APPROVED** [review]

**Pull request merged**